CommonLibF4/include/RE/Bethesda/BSExtraData.h

```diff
@@ -619,7 +619,11 @@ namespace RE
 			return !((stl::to_underlying(a_type) - 11) & ~0x22u) && a_type != EXTRA_DATA_TYPE::kLeveledCreature;
 		}
 
-		void CreateFlags() { _flags = calloc<std::uint8_t>(N); }
+		void CreateFlags()
+		{
+			_flags = calloc<std::uint8_t>(N);
+			memset(_flags, 0, N);
```

Comment:
```suggestion
			std::memset(_flags, 0, N);
```
Please resolve all `std` namespaces

Reply:
Done.